content/4_Gold/BFS.mdx

```diff
@@ -92,6 +92,11 @@ v = q.get()      # v = 1, q = [2]
 v = q.get()      # v = 2, q = []
 v = q.get()      # Code waits forever, creating TLE error.
 ```
+<Warning>
+
+Python's queue.Queue() uses Locks to maintain a threadsafe synchronization, thereby making it quite slow. To avoid TLE in many questions, rather use collections.deque() for a faster queue data structure.
```

Comment:
```suggestion
Python's `queue.Queue()` uses Locks to maintain a threadsafe synchronization, thereby making it quite slow. To avoid TLE in many questions, rather use `collections.deque()` for a faster queue data structure.
```